src/reference/informative/informC04_component_reference.rst

```diff
@@ -8,4 +8,189 @@
+
+        See CellML syntax
+
+      .. code-block:: xml
+
+        <model name="SurbitonNeighbourhood">
+          ...  
+          <component name="WomenInTheNeighbourhood" />
+          <component name="MenInTheNeighbourhood" />
+
+          <!-- Throughout the importing model (i.e.: this model), the imported
+              items are referred to by their local name attribute ("BarbaraGood" etc), 
+              not the name they are called within their imported family units ("wife"). -->
+          <encapsulation>
+            <component_ref component="WomenInTheNeighbourhood">
+              <component_ref component="BarbaraGood" />
+              <component_ref component="MargotLeadbetter" />
+            </component_ref>
+            <component_ref component="MenInTheNeighbourhood">
+              <component_ref component="TomGood" />
+              <component_ref component="JerryLeadbetter" />
+            </component_ref>
+          </encapsulation>
+        </model>
+
+    This particular encapsulation structure means that the women (Barbara and Margot) are essentially unable to have any contact with the men (Tom and Jerry) even though their original components in the models from which they were imported *were* able to access one another.
+
+    Note also that these locality naming-calling rules are be applied through multiple generations of importing.
+    Since The Good Life is a TV show, there are actors who play the roles of each of the characters. 
+    This could be reflected by using another generation of imports within the two family files like this:
+
+    .. code::
+
+        model: SurbitonNeighbourhood
+          ├─ component: WomenInTheNeighbourhood
+          │   ├─ component: BarbaraGood <╴╴╴╴╴╴╴╴╴╴╴╴╴┐
+          │   └─ component: MargotLeadbetter <╴╴╴╴╴╴┐ ╷
+          └─ component: MenInTheNeighbourhood       ╷ ╷
+              ├─ component: TomGood <╴╴╴╴╴╴╴╴╴╴╴╴╴┐ ╷ ╷
+              └─ component: JerryLeadbetter <╴╴╴┐ ╷ ╷ ╷
+                                                ╷ ╷ ╷ ╷
+                                      imported components
+                                                ╵ ╵ ╵ ╵
+                # In LeadbetterFamily.cellml:   ╵ ╵ ╵ ╵
+                model: TheLeadbetterFamilyUnit  ╵ ╵ ╵ ╵
+        ┌╴╴╴╴╴╴╴╴> ├─ component: husband ╴╴╴╴╴╴╴┘ ╵ ╵ ╵
+        ╷ ┌ ╴╴╴╴╴> └─ component: wife ╴╴╴╴╴╴╴╴╴╴╴╴┼╴┘ ╵
+        ╷ ╷                                       ╵   ╵
+        ╷ ╷     # In GoodFamily.cellml:           ╵   ╵
+        ╷ ╷     model: TheGoodFamilyUnit          ╵   ╵
+        ╷ ╷ ┌╴╴╴> ├─ component: husband ╴╴╴╴╴╴╴╴╴╴┘   ╵
+        ╷ ╷ ╷ ┌╴> └─ component: wife ╴╴╴╴╴╴╴╴╴╴╴╴╴╴╴╴╴┘
+        ╷ ╷ ╷ ╷
+      imported components
+        ╵ ╵ ╵ ╵
+        ╵ ╵ ╵ ╵  # In CastOfCharacters.cellml:
+        ╵ ╵ ╵ ╵  model: 
+        ╵ ╵ ╵ └╴╴╴├─ component: FelicityKendal
+        ╵ ╵ └╴╴╴╴╴├─ component: RichardBriers
+        ╵ └╴╴╴╴╴╴╴├─ component: PenelopeKeith
+        └╴╴╴╴╴╴╴╴╴└─ component: PaulEddington
```

Comment:
In case we decide to use plain lines:

```
  model: SurbitonNeighbourhood
    ├─ component: WomenInTheNeighbourhood
    │   ├─ component: BarbaraGood <─────────────┐
    │   └─ component: MargotLeadbetter <──────┐ │
    └─ component: MenInTheNeighbourhood       │ │
        ├─ component: TomGood <─────────────┐ │ │
        └─ component: JerryLeadbetter <───┐ │ │ │
                                          │ │ │ │
                                imported components
                                          │ │ │ │
          # In LeadbetterFamily.cellml:   │ │ │ │
          model: TheLeadbetterFamilyUnit  │ │ │ │
  ┌────────> ├─ component: husband ───────┘ │ │ │
  │ ┌──────> └─ component: wife ────────────┼─┘ │
  │ │                                       │   │
  │ │     # In GoodFamily.cellml:           │   │
  │ │     model: TheGoodFamilyUnit          │   │
  │ │ ┌───> ├─ component: husband ──────────┘   │
  │ │ │ ┌─> └─ component: wife ─────────────────┘
  │ │ │ │
imported components
  │ │ │ │
  │ │ │ │  # In CastOfCharacters.cellml:
  │ │ │ │  model:
  │ │ │ └── ├─ component: FelicityKendal
  │ │ └──── ├─ component: RichardBriers
  │ └────── ├─ component: PenelopeKeith
  └──────── └─ component: PaulEddington
```

Reply:
I can do a global search and replace if we decide to ditch the dashed ones.  There were several options (including different weights), but more important to get the content in for now, I think.

Reply:
Agreed.